Snowflake.Data/Core/Session/SFSessionProperty.cs

```diff
@@ -90,7 +90,9 @@ internal enum SFSessionProperty
         [SFSessionPropertyAttr(required = false, defaultValue = "false")]
         DISABLEQUERYCONTEXTCACHE,
         [SFSessionPropertyAttr(required = false)]
-        CLIENT_CONFIG_FILE
+        CLIENT_CONFIG_FILE,
+        [SFSessionPropertyAttr(required = false, defaultValue = "false")]
+        ALLOWUNDERSCORESINHOST

```

Comment:
Do we need to document this in the readme?

Reply:
It has been documented in the readme.

Reply:
my bad, didn't notice that file